### DISCUSSION TITLE
[BUG] ...

### DISCUSSION THREAD

**Discussion by hacker0x01hackerone** [projectdiscovery/nuclei#6433] [General]
### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

![Image](https://github.com/user-attachments/assets/e4b441b0-0be9-46c7-a1f7-32df727b324a)
http templates doesn't work, I've already tried to reset, I've reinstalled nuclei, but I still get this error message for this command nuclei -l ./cmsub.txt -t http
[ERR] Could not find template '[iis-shortname] [http] [info] https://www.fixtectools.com/*~1*/a.aspx': no templates found for path
[ERR] Could not find template '[email-extractor] [http] [info] https://www.fixtectools.com/ ["fixtec@fixtectools.com"]': could not find template file: no such path found: [email-extractor] [http] [info] https://www.fixtectools.com/ ["fixtec@fixtectools.com"]
[ERR] Could not find template '[aws-detect:aws-cloudfront] [http] [info] https://www.fixtectools.com/': could not find template file: no such path found: [aws-detect:aws-cloudfront] [http] [info] https://www.fixtectools.com/
[ERR] Could not find template '[ssl-issuer] [ssl] [info] www.fixtectools.com:443 ["Amazon"]': could not find file: open /home/kali/.local/nuclei-templates/[ssl-issuer] [ssl] [info] www.fixtectools.com:443 ["Amazon"]: no such file or directory
[ERR] Could not find template '[tech-detect:animate.css] [http] [info] https://www.fixtectools.com/': could not find template file: no such path found: [tech-detect:animate.css] [http] [info] https://www.fixtectools.com/
[ERR] Could not find template '[missing-sri] [http] [info] https://www.fixtectools.com/ ["//jprorwxhrlmjli5q-static.micyjz.com/phoenix/optimization/pPpfWAHaRhdZ/0/speedLazyLoad4.min.js?_=1730107975589"]': could not find template file: no such path found: [missing-sri] [http] [info] https://www.fixtectools.com/ ["//jprorwxhrlmjli5q-static.micyjz.com/phoenix/optimization/pPpfWAHaRhdZ/0/speedLazyLoad4.min.js?_=1730107975589"]
[ERR] Could not find template '[iis-shortname] [http] [info] https://www.fixtectools.com/N0t4xist*~1*/a.aspx': no templates found for path
[ERR] Could not find template '[tech-detect:font-awesome] [http] [info] https://www.fixtectools.com/': could not find template file: no such path found: [tech-detect:font-awesome] [http] [info] https://www.fixtectools.com/
[ERR] Could not find template '[tech-detect:google-font-api] [http] [info] https://www.fixtectools.com/': could not find template file: no such path found: [tech-detect:google-font-api] [http] [info] https://www.fixtectools.com/

I looked at the templates, they are there, but for some reason it doesn't recognize it and the url in the error message has nothing to do with what I want to check, it looks like only the http templates don't work, the others do

### Expected Behavior

that it will work because it has worked properly so far

### Steps To Reproduce

nuclei -l cmsub.txt -t http

### Environment

- OS: Kali Linux 2024.1(arm64l
- Nuclei: 3.3.5
- Go: go1.23.2 linux/arm64

**Comment by Mzack9999** [comment]
You need to specify the protocol with `-pt` as `-t` is used for paths:

```
nuclei -pt http -l targets.txt
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
